### PR TITLE
Milestone v230131

### PR DESCRIPTION
**Pull request by sultanmyrza**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201016280880500/1203926131240409) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/app/features/wallets/buy-num/buy-num.page.html [hunk 1]
```diff
@@ -28,7 +26,7 @@
                 name="alert-circle-outline"
               ></ion-icon>
             </div>
-            <div class="subtitle-text" *ngIf="!first">
+            <div class="subtitle-text">
               {{ product.inAppProduct.price }}
             </div>
           </div>
```
Comment (sultanmyrza) [inline src/app/features/wallets/buy-num/buy-num.page.html:32]: ChatGPT [comment](https://github.com/numbersprotocol/capture-lite/pull/2521#discussion_r1089842500)

File: src/app/features/wallets/wallets.page.ts [hunk 2]
```diff
@@ -85,6 +96,73 @@ export class WalletsPage {
...
+        first(),
+        concatMap(privateKey => this.copyToClipboard(privateKey)),
+        untilDestroyed(this)
+      )
+      .subscribe();
+  }
+
   // eslint-disable-next-line class-methods-use-this
   onBuyNumBtnClicked() {
     Browser.open({
```
Comment (sultanmyrza) [inline src/app/features/wallets/wallets.page.ts]: ChatGPT [comment](https://github.com/numbersprotocol/capture-lite/pull/2525/commits/5a35c2c32ab113c43a0ed244d389f5e03784f3d8#r1089946522)

**Comment by sultanmyrza** [comment]
@tammyyang, as you can see lazy login [PR](https://github.com/numbersprotocol/capture-lite/pull/2529) has most ChatGPT comments but that PR was reverted when it didn't pass QA testing because it needs some work from iframe side to be complete. 

So this mileston has only 2 ChatGPT comments only.

**Comment by tammyyang** [comment]
All looks good to me.

**Review by tammyyang — APPROVED** [review]
All done

**Pull request merged**